src/editor.cpp

```diff
@@ -20,7 +20,7 @@ int main()
 	LEI_TRACE("Initialized Logger");
 
 	// loops until render loop ends
-	Application app;
+	Application& app = Application::GetInstance();
```

Comment:
I know this was to make our app thread safe, but I'll try to get rid of our multi threading in audio for performance reasoning, just so we all know.